api/graphql/schema.graphql

```diff
@@ -478,6 +483,15 @@ type PoolCandidate {
   signature: String
   submittedSteps: [ApplicationStep!] @rename(attribute: "submitted_steps")
   screeningQuestionResponses: [ScreeningQuestionResponse] @hasMany
+  minimumCriteria: ApplicationCriteria @rename(attribute: "minimum_criteria")
+
+  # experiences for meeting minimum criteria
+  awardExperiencesCriteria: [AwardExperience] @belongsToMany
+  communityExperiencesCriteria: [CommunityExperience] @belongsToMany
+  educationExperiencesCriteria: [EducationExperience] @belongsToMany
+  personalExperiencesCriteria: [PersonalExperience] @belongsToMany
+  workExperiencesCriteria: [WorkExperience] @belongsToMany
+  experiencesCriteria: [Experience] #collection of above five
```

Comment:
Could we rename this to minimumCriteriaExperiences to make its link to minimumCriteria especially obvious? Perhaps with comments on these two fields as well.